manager/manifests/fluentd.yaml

```diff
@@ -168,10 +168,15 @@ metadata:
   name: fluentd
   namespace: default
 spec:
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
   template:
     metadata:
       labels:
         app: fluentd
+        hallo: dummy
     spec:
       serviceAccountName: fluentd
       priorityClassName: fluentd
```

Comment:
(I'm putting this comment here because GitHub won't let me put it lower): did you try testing the rolling update after changing the container image? If not I can, just checking!

Reply:
yes i tried the rolling update and it looks like it works.